addons/hr/models/hr_department.py

```diff
@@ -64,6 +64,20 @@ def create(self, vals):
             department.message_subscribe(partner_ids=manager.user_id.partner_id.ids)
         return department
 
+    def get_formview_action(self, access_uid=None):
+        res = super().get_formview_action(access_uid)
+        if self.env.context.get('open_employees_kanban', False):
+            res.update({
+                'name': self.name,
+                'res_model': 'hr.employee',
+                'view_type': 'kanban',
+                'view_mode': 'kanban',
+                'views': [(False, 'kanban'), (False, 'form')],
+                'context': {'searchpanel_default_department_id': self.id},
+                'res_id': False,
+            })
+        return res
+
```

Comment:
I think this should be done in `hr_attendance` and only for Kiosk users (`hr_attendance.group_hr_attendance_kiosk`), same for the context key in the view

Reply:
Done, thanks :-)